### PR TITLE
Vulcan: Problem List: Metadata

### PR DESCRIPTION
**Pull request by ianrohde**
## Issue

Update boilerplate NextJS metadata with text from Max.

> [!NOTE]
> There will likely be a follow-up issue to allow overriding the text for these values. https://github.com/iFixit/ifixit/issues/51218#issuecomment-1861933848

## CR/QA

Confirm we're seeing the correct tags on the Vercel dev preview:

https://react-commerce-a9gsmmpy1-ifixit.vercel.app/Troubleshooting/Dryer

Title:
```
Troubleshooting Problems for ${device} - iFixit
```

Description:
```
Is your ${device} not working? You are not alone. View common problems and explore potential solutions. Learn the steps you can take to troubleshoot and fix the problem yourself.
```

Closes https://github.com/iFixit/ifixit/issues/51218

### CODE REVIEW

**Comment by jordycosta** [comment]
Should the device be interpolated here for the description? I currently see ${device}

meta

**Review by jarstelfox — DISMISSED** [review]
CR 🌵

**Comment by ianrohde** [comment]
dev_block 🔴

File: frontend/app/(defaultLayout)/Troubleshooting/[device]/page.tsx [hunk 1]
```diff
@@ -76,9 +76,9 @@ export async function generateMetadata({
    const ifixitOrigin = getiFixitOrigin(nextHeaders);
    const canonicalUrl = `${ifixitOrigin}/Troubleshooting/${device}`;
 
-   const metaTitle = `Troubleshooting Collection for ${device} - iFixit`;
+   const metaTitle = `Troubleshooting Problems for ${device} - iFixit`;
    const metaDescription =
-      'Uses the App Router to render a page with a dynamic path.';
+      'Is your ${device} not working? You are not alone. View common problems and explore potential solutions. Learn the steps you can take to troubleshoot and fix the problem yourself.';
```
Comment (jordycosta) [inline frontend/app/(defaultLayout)/Troubleshooting/[device]/page.tsx:81]:
```suggestion
     `Is your ${device} not working? You are not alone. View common problems and explore potential solutions. Learn the steps you can take to troubleshoot and fix the problem yourself.`;
```

Comment (jordycosta) [reply frontend/app/(defaultLayout)/Troubleshooting/[device]/page.tsx:81]: I think this should resolve that issue

Comment (ianrohde) [reply frontend/app/(defaultLayout)/Troubleshooting/[device]/page.tsx:81]: :doh:

**Review by jarstelfox — DISMISSED** [review]
CR 🌵

**Review by jarstelfox — APPROVED** [review]
CR 🌵

**Comment by jordycosta** [comment]
QA 🟢 

meta

**Pull request merged**